src/paint.js

```diff
@@ -237,8 +237,8 @@ PaintEditorMorph.prototype.buildEdits = function () {
         function () {
             if (myself.paper.undoBuffer.length > 0) {
                 myself.ide.confirm(
-                    'This will erase your current drawing.\n' +
-                    'Are you sure you want to continue?',
+                    localize('This will erase your current drawing.\n') +
```

Comment:
hmmm.... I don't think these strings need to be explicitly localized at all here. When I look at the code for the ```IDE_Morph >> confirm```  method it already has a call to ```localize``` in it.

Reply:
@jmoenig 
Yes, but the issue here is the same as the previous comment. I know ide.confirm has already a call to localize, but here we have again message made up with different strings. If we want that the translation dictionary was the same that the code dictionary (the same strings), I have to call a "localize" for each shorter string.